manifests/workflow-controller-cluster-wide-rbac.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Does workflow repository has these rbac yaml files? If it has, we should only reference them in the doc.

Reply:
There are cluster-scope rbac settings available at argo-workflows, but in their original form they will not be bound to argo namespace.
What is possible is to create kustomization.yaml like this:
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

resources:
  - https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrole.yaml
  - https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrolebinding.yaml

patches:
  - patch: |-
      - op: add
        path: /subjects/0/namespace
        value: argo
    target:
      kind: ClusterRoleBinding
```

Reply:
@whynowy 
There are cluster-scope rbac settings available at argo-workflows, but in their original form they will not be bound to argo namespace.
What is possible is to create kustomization.yaml like this:
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

resources:
  - https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrole.yaml
  - https://raw.githubusercontent.com/argoproj/argo-workflows/master/manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrolebinding.yaml

patches:
  - patch: |-
      - op: add
        path: /subjects/0/namespace
        value: argo
    target:
      kind: ClusterRoleBinding
```